### ISSUE TITLE
404 error when the destination is invalid

### ISSUE THREAD

**Issue by sudheshna-donthineni** [avantifellows/portal-frontend#13]
If the `purpose` in the query params is not either `plio` or `liveclass`, then the `redirectToDestination` method will return false.
For the backend, there will be an error log in AWS Lambda. 
Need to create a component for the user to know.

**Comment by pritamps** [comment]
This can actually be a general 404 page, not only for this error, though this error should redirect to the 404 page, of course

**Comment by sudheshna-donthineni** [comment]
Yeah, I was thinking the same thing. A 404 component that is reusable.

**Comment by Akshanshkaushal** [comment]
@sudheshna-donthineni  I would love to work upon this issue. Could you please assign this to me ?
